splunk/splunk_container_template.yaml

```diff
@@ -2,21 +2,20 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: {{ .PodName}}
+  name: { { .PodName } }
```

Comment:
these changes will make this file no longer function as a proper go template. Perhaps we should change the extension for this file because its not a legit YAML file.

Reply:
`.templ` or `.got` I suppose - unless you manage to find some other convention for naming Go template files.

Reply:
With Jinja2 to convention was to use `.j2` so for example `foo.bar.j2` would be a template for generating `foo.bar`

Reply:
fixed, chose to use `.tmpl`.